core/src/main/java/org/mule/expression/DefaultExpressionManager.java

```diff
@@ -604,6 +604,14 @@ public void initialise() throws InitialisationException
     {
         MVELExpressionLanguage mel = new MVELExpressionLanguage(muleContext);
         mel.initialise();
+        try
```

Comment:
It's not. It's related to the comment on MVELExpressionLanguage.java. Will review as said before